Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/TestingSyncUpDetail.tutorial

```diff
@@ -46,9 +46,7 @@
       
       @Step {
         Assert how the state changes after sending the action. In particular, the `syncUp` data
-        inside the `edit` case of the destination should change. We can use the special 
-        ``ComposableArchitecture/PresentationState/subscript(case:)-7uqte`` defined on
-        `$destination` to modify the data in a particular case.
+        inside the `edit` case of the destination should change.
```

Comment:
This subscript syntax isn't being used here:
https://github.com/pointfreeco/swift-composable-architecture/blob/af38644542b1310c0bb023a5af2545beb8854009/Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/06-SyncUpDetail/TestingSyncUpDetail-01-code-0005.swift#L24